apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/UpdateQuestionId.tsx

```diff
@@ -40,7 +40,15 @@ export default function UpdateQuestionId({
       updateQuestion(questionIdx, { id: prevValue });
       toast.error("ID should not be empty.");
       return;
-    } else if (currentValue === "source" || currentValue === "suID" || currentValue === "userId") {
+    } else if (
+      currentValue === "source" ||
+      currentValue === "suID" ||
```

Comment:
why are the reserved words for both cases different? shouldn't they be the same?
Also if possible I would also prefer a Array.includes() listing which is shorter :-)